portfolio/src/main/webapp/script.js

```diff
@@ -55,7 +41,29 @@ function createCommentElement(comment) {
   const contentElement = document.createElement('span'); 
   contentElement.innerText = comment.content;
 
+  
+  const imgElement = document.createElement('img');
+  if(comment.imageUrl.substr(1,10) == "cloudshell"){
+    imgElement.src = comment.imageUrl.substr(11);
+  }
+  else
+    imgElement.src = comment.imageUrl;
```

Comment:
Nit: Add curly braces for each branch, and add whitespaces between `if` and `(`, `)` and `{`

```
if (condition) {
  branchA;
} else {
  branchB;
}